testing/tests/configs_tree_view.py

```diff
@@ -34,202 +34,428 @@
                 - :ref:`exhale_struct_structnested_1_1two_1_1params`
                     - :ref:`exhale_union_unionnested_1_1two_1_1params_1_1four__bytes`
             - :ref:`exhale_union_unionnested_1_1four__bytes`
+        - :ref:`namespace_special`
```

Comment:
I really wasn't sure about this file, but since the doxygen of the nested_cpp project changed, all of the HTML also changed. 
I'm not too well versed in what's supposed to happen, so I got them to pass by adding the output as I was seeing it.